docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -67,16 +67,29 @@ NOTE: The terminal is out of use while the Core Server is running.
 
 [id='{context}-published-repositories-running-the-mobile-and-portal-applications']
 [discrete]
-== Running the Mobile and Portal Applications
+== Starting the Portal and Mobile Applications
 
-Open a new terminal and execute the following command in folder `raincatcher-portal/` and also in folder `raincatcher-mobile/`:
+1&#46; Start the Portal application as follows:
 
 [source,bash]
 ----
+cd raincatcher-portal/
 npm run start
 ----
 
-Both the Mobile and Portal Applications open automatically in a browser.
+2&#46; Start the Mobile application as follows:
```

Comment:
I'm not fan of unicode format - what is that? Does it render properly?

Reply:
yeah, this is kind of a hack that I don't like. it renders as 2.

the reason for this is that if you insert [source,bash] between steps in an ordered list, or any other code snippet or content, it breaks the numbering when the list is rendered. it starts numbering over. so what should be step 2 becomes step 1.

my solution is to 'hard-code' the numbering and force it by inserting that entity.

if you don't want to keep this, and that's fine, perhaps a solution would be to say "First, start the..." and then "Next, start the..."